test/micro/org/openjdk/bench/vm/compiler/CodeCacheStress.java

```diff
@@ -0,0 +1,445 @@
+                + "    int padAH = 0;"
+                + "    int padAI = 0;"
+                + "    int padAJ = 0;"
+                + "    int padAK = 0;"
+                + "    int padAL = 0;"
+                + "    int padAM = 0;"
+                + "    int padAN = 0;"
+                + "    int padAO = 0;"
+                + "    int padAP = 0;"
+                + "    int padAQ = 0;"
+                + "    int padAR = 0;"
+                + "    int padAS = 0;"
+                + "    int padAT = 0;"
+                + " "
+                + "    int instB = 0;"
+                + " "
+                + "    int padBA = 0;"
+                + "    int padBB = 0;"
+                + "    int padBC = 0;"
+                + "    int padBD = 0;"
+                + "    int padBE = 0;"
+                + "    int padBF = 0;"
+                + "    int padBG = 0;"
+                + "    int padBH = 0;"
+                + "    int padBI = 0;"
+                + "    int padBJ = 0;"
+                + "    int padBK = 0;"
+                + "    int padBL = 0;"
+                + "    int padBM = 0;"
+                + "    int padBN = 0;"
+                + "    int padBO = 0;"
+                + "    int padBP = 0;"
+                + "    int padBQ = 0;"
+                + "    int padBR = 0;"
+                + "    int padBS = 0;"
+                + "    int padBT = 0;"
+                + " "
+                + "    int instC = 0;"
+                + " "
+                + "    int padCA = 0;"
+                + "    int padCB = 0;"
+                + "    int padCC = 0;"
+                + "    int padCD = 0;"
+                + "    int padCE = 0;"
+                + "    int padCF = 0;"
+                + "    int padCG = 0;"
+                + "    int padCH = 0;"
+                + "    int padCI = 0;"
+                + "    int padCJ = 0;"
+                + "    int padCK = 0;"
+                + "    int padCL = 0;"
+                + "    int padCM = 0;"
+                + "    int padCN = 0;"
+                + "    int padCO = 0;"
+                + "    int padCP = 0;"
+                + "    int padCQ = 0;"
+                + "    int padCR = 0;"
+                + "    int padCS = 0;"
+                + "    int padCT = 0;"
+                + " "
+                + "    int instD = 0;"
+                + " "
+                + " "
+                + "   public Integer get(Map m, String k, Integer depth) { "
+                + "       if (depth > 0) {"
+                + "         instA += ((depth % 2) + staticA);"
+                + "         return (Integer) m.get(k) + get2(m, k, --depth);"
+                + "       } else {"
+                + "         setA(depth);"
+                + "         return (Integer) m.get(k)+ 10;"
+                + "       }"
+                + "   }"
+                + " "
+                + "   public Integer get2( Map m, String k, Integer depth) { "
```

Comment:
Here and later `get2( ` has redundant spaces after the parenthesis.

Reply:
Thanks, fixed in the next rev.